pytorch_toolkit/object_detection/face-detection/face-detection-0200/config.py

```diff
@@ -128,7 +132,7 @@
 checkpoint_config = dict(interval=1)
 # yapf:disable
 log_config = dict(
-    interval=100,
+    interval=10,
```

Comment:
What's the reason of this change?

Reply:
too long to wait until first loss value appears :)

Reply:
Makes sense 🤣